core/src/bundle/usCoreBundleContext.cpp

```diff
@@ -83,7 +83,6 @@ CoreBundleContext::~CoreBundleContext()
 
 void CoreBundleContext::Init()
 {
-  US_INFO("initializing");
```

Comment:
Can we keep this and the messages below as diagnostic message? I found them quite useful during debugging.